steam_wrapper/steam_wrapper.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python3
+import os
+import os.path
+import sys
+import shutil
+import errno
+import fnmatch
+import subprocess
+
+STEAM_PATH = "/app/bin/steam"
+STEAM_ROOT = os.path.expandvars("$HOME/.var/app/com.valvesoftware.Steam")
+
+def prompt():
+    subprocess.check_call(["zenity", "--question",
+                            ("--text="
+                            "This is com.valvesoftware.Steam cloud sync repair. "
+                            "If you have conflicting local and cloud data for "
+                            "your game, this may result in partial loss of your "
+                            "cloud data. If you instead prefer ensuring cloud data "
+                            "persists, please relocate your "
+                            "~/.var/app/com.valvesoftware.Steam/data/Steam "
+                            "to a secure location, "
+                            "remove ~/.var/app/com.valvesoftware.Steam "
+                            "and put Steam data directory back to avoid needing to "
+                            "re-download games")])
+
+def ignored(name, patterns):
+    for pattern in patterns:
+        if fnmatch.fnmatch(name, pattern):
+            return True
+    else:
+        return False
+
+def filter_names(root, names, patterns):
+    _names = []
+    for name in names:
+        if not ignored(os.path.join(root, name), patterns):
+            _names.append(name)
+    return _names
+
+def copytree(source, target, ignore=None):
```

Comment:
I imagine `shutil.copytree()` wasn't the behavior you wanted?

Reply:
It has a deficiency that if target directory exists, it raises exception and no way to avoid it. There's an open PR about this which might eventually get into a later Python release.

Reply:
Specifically https://github.com/python/cpython/pull/8792